ann/docker-compose.yml

```diff
@@ -7,18 +7,30 @@ services:
     ports:
       - "3000:3000"
     depends_on:
-      - mysql
+      mysql:
+        condition: service_healthy
     environment:
       MYSQL_HOST: mysql
       MYSQL_PORT: 3306
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+      SECRET_KEY: ${SECRET_KEY}
     volumes:
-      - ".:/app"
-      - "/app/node_modules"
+      - "./src:/app/src"
```

Comment:
Only mount ./src. Any file changes in the base route `./` should require a restart anyways.